EVGEN/AliGenExtFile.cxx

```diff
@@ -298,8 +338,32 @@ void AliGenExtFile::CdEventFile()
 // CD back to the event file
   AliRunLoader::Instance()->CdGAFile();
 }
+//__________________________________________________________
+Bool_t AliGenExtFile::MultiplicityTrigger(AliStack *stack){
+  Int_t nTracks  = stack->GetNtrack();
+  AliInfo(Form("n Tracks = %i",nTracks));
+  if(nTracks>fMultCut){return kTRUE;}
```

Comment:
Would it not be interesting to apply not only a high limit but also a low limit?

Reply:
AliGenExtFile::MultiplicityTrigger() and AliGenExtFile::PtTrigger() are mainly meant to be two easy-to-use methods which can also provide an example for the user to implement his/her own trigger with fUserTrigger(stack)

Reply:
OK